### PR TITLE
Basic spaceflight

### PR DESCRIPTION
**Pull request by SuperShadowPlay**
Created the base game mechanic of spaceflight.

### CODE REVIEW

File: keyboard.lua [hunk 1]
```diff
@@ -10,24 +10,53 @@ keyboard.key_map = {
...
     end
     if love.keyboard.isDown('s') then
+        obj.body:applyForce(ratiox * movementForce * -0.25, ratioy * movementForce * -0.25)
+    end
+
+    -- Steer spaceship left and right
+    local angVel = player.body:getAngularVelocity()
+    local maxAngVel = 2
+    if angVel > -maxAngVel and angVel < maxAngVel then
+        if love.keyboard.isDown('a') then
```
Comment (jaythomas) [inline keyboard.lua]: Consider using [`love.keyboard.isScancodeDown()`](https://love2d.org/wiki/love.keyboard.isScancodeDown) instead. I don't have a qwerty keyboard so these bindings don't make sense trying to play on my keyboard but the scancode reference the physical switch locations regardless of what letter is mapped to that key in the OS.

**Comment by jaythomas** [comment]
For the engine.mp3 you might want to consider a shortened version of the clip so that it loops better and is a much smaller filesize.

**Comment by jaythomas** [comment]
This is looking good. Controls are working now for me too. Did you want to squash this in before moving on to the next thing?

**Comment by SuperShadowPlay** [comment]
Yeah I think that's probably the way to go. It's in a "playable" state so that anyone who takes a look at it will at least be able to pilot the cube around a bit.

**Pull request merged**